src/distances.jl

```diff
@@ -34,3 +26,23 @@ with `T` of appropriate type for members of the set.
 function distance_to_set end
 
 distance_to_set(::AbstractDistance, v, s) = distance_to_set(DefaultDistance(), v, s)
+
+"""
+    EpigraphViolationDistance
+
+Distance used when `v ∈ s` can be expressed as a single scalar function `f(v) ≤ 0`.
+The distance is expressed as `max(f(v), 0)`.
+"""
+struct EpigraphViolationDistance <: AbstractDistance end
+
+"""
+    NormedEpigraphDistance{p}
+
+Distance used when `v ∈ s` can be expressed as a several functions `f_i(v) ≤ 0 ∀ i ∈ I`.
+The distance is expressed as `norm([max(f_i(v), 0) ∀ i ∈ I], n)` with `p` a norm value (2, 1, Inf).
+"""
+struct NormedEpigraphDistance{p} <: AbstractDistance end
+
+# _norm_set(::NormedEpigraphDistance{1}) = MOI.NormOneCone
```

Comment:
Are these comments expected?

Reply:
nope, removed, I thought I would have needed these, in the end nope